README.md

```diff
@@ -453,6 +454,9 @@ rest.push.admin.publishAsync(recipient, payload, , new CompletionListener() {
  });
 ```
 
+Note that only JsonElement, String, Boolean, Character, Number, and JsonUtilsObject are supported by JsonUtils.
+If an unsupported type is added JsonUtils will throw JsonParseException!
```

Comment:
I'm not sure if that exclamation mark at the end is necessary :sweat_smile: 
```suggestion
If an unsupported type is added JsonUtils will throw JsonParseException.
```

Reply:
here it is https://github.com/ably/ably-java/pull/729/commits/1820e44c901e011075d91c7edd249a237f3ea0ab